angularSTEP/src/app/view-profiles/view-profiles.component.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  AngularFirestore,
+  AngularFirestoreCollection,
+} from '@angular/fire/firestore';
+import {Component, OnInit} from '@angular/core';
+
+import {AngularFireAuth} from '@angular/fire/auth';
+import {Observable} from 'rxjs';
+import {Router} from '@angular/router';
+import {User} from '../user';
```

Comment:
Spacing.

Reply:
Thank you again for pointing this out. The problem with imports, like here, is that the Google Style guide actually changes them to have no spaces at all. I was confused why all my commits always had no spaces between the {} even when I would explicitly change them. We might want to actually switch to having no spaces for imports if that is what the Google Style guide wants.